rotkehlchen/tests/unit/decoders/test_main.py

```diff
@@ -224,3 +224,81 @@ def test_simple_erc20_transfer(
             extra_data=None,
         ),
     ]
+
+
+@pytest.mark.parametrize('ethereum_accounts', [[
+    '0x4bBa290826C253BD854121346c370a9886d1bC26',
+    '0x38C3f1Ab36BdCa29133d8AF7A19811D10B6CA3FC',
+]])
+def test_eth_transfer(
+        database,
+        ethereum_accounts,
+        evm_transaction_decoder,
+):
+    """
+    Data taken from
+    https://etherscan.io/tx/0x8caa7df2ebebfceb98207605e64691202b9e7498c3cccdbccb41c1600cf16e65
+    """
```

Comment:
I believe we have some test for those but can you check also the deposit and the spend?